torch/quantization/fake_quantize.py

```diff
@@ -156,17 +156,25 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                                                       qscheme=torch.per_tensor_affine,
                                                       reduce_range=True)
 def disable_fake_quant(mod):
-    if type(mod) == FakeQuantize:
+    if type(mod) == FakeQuantize or\
+        isinstance(mod, torch.jit.RecursiveScriptModule)\
+            and mod.original_name == 'FakeQuantize':
         mod.disable_fake_quant()
 
 def enable_fake_quant(mod):
-    if type(mod) == FakeQuantize:
+    if type(mod) == FakeQuantize or\
+        isinstance(mod, torch.jit.RecursiveScriptModule)\
+            and mod.original_name == 'FakeQuantize':
         mod.enable_fake_quant()
 
 def disable_observer(mod):
-    if type(mod) == FakeQuantize:
+    if type(mod) == FakeQuantize or\
+        isinstance(mod, torch.jit.RecursiveScriptModule)\
+            and mod.original_name == 'FakeQuantize':
```

Comment:
is there a way to get the full qualified path?

Reply:
You mean like `torch.quantization.FakeQuantize`? I wasn't able to find a way to do that.

Reply:
yeah, can you try to see if this works? https://github.com/pytorch/pytorch/blob/master/torch/_jit_internal.py#L736

Reply:
oh this one:https://codebrowser.bddppq.com/pytorch/pytorch/torch/csrc/jit/python/script_init.cpp.html#1046

you can probably get this by `m._c. qualified_name`